### ISSUE TITLE
What is the format of the R-CNN training data?

### ISSUE THREAD

**Issue by wuqiangch** [BVLC/caffe#873]
the paper is "Rich feature hierarchies for accurate object detection and semantic segmentation"
I want to use the method to fine tune on my own database.
what i only do is to produce files like 'window_file_voc_2012_train.txt";
/path/to/VOC2012/VOCdevkit/VOC2012/JPEGImages/2008_000008.jpg -> picture's name
3 channels
442 ->rows
500 ->cols
2753 -> the number of the boxes
13 1.000 52 86 470 419 -> label, overlap rate,x1,y1,x2,y2 
15 1.000 157 43 288 166
Anyone can tell me wheather the explansion is right or any other files I must need?
